tools/release/release_utils.sh

```diff
@@ -70,6 +70,22 @@ function publishImages {
     docker logout
 }
 
+function publishPackages{
```

Comment:
There is a missing space after the function name publishPackages, it should be written as` publishPackages {`